adapters/Chartboost/ChartboostAdapter/GADMAdapterChartboostSingleton.m

```diff
@@ -45,56 +36,20 @@ + (nonnull GADMAdapterChartboostSingleton *)sharedInstance {
   return sharedInstance;
 }
 
-- (nonnull instancetype)init {
-  self = [super init];
-  if (self) {
-    _completionHandlers = [[NSMutableArray alloc] init];
-  }
-  return self;
-}
-
 - (void)startWithAppId:(nonnull NSString *)appId
```

Comment:
This class now loses its use since we can call `[Chartboost startWithAppId:]` directly. 
Please delete this class and update the other classes to call `[Chartboost startWithAppId:]` directly.

Reply:
Makes sense 👍